### DISCUSSION TITLE
I keep on getting this error when trying to setup the progam even tho i have that file

### DISCUSSION THREAD

**Discussion by Kimberlyrios1111** [nianeyna/ao3downloader#144] [Q&A]
i keep on getting that after i do the step pip install -r requirements.txt 
my setup is a m1 macbook air and i am running it in the terminal 
do you perhaps have a solution to the problem? 
the version of python i have is : Python 3.11.4
an image for reference 
<img width="1440" alt="Screenshot 2024-06-09 at 4 27 10 PM" src="https://github.com/nianeyna/ao3downloader/assets/147880405/c364192b-4b01-4f05-84e8-73e6f5fdbc2e">

**Comment by nianeyna** [comment]
Did you cd into the correct folder before running the rest of the commands? it looks like you're running them from your home directory. For reference the instructions for getting to the correct folder in your terminal are:

> You can do this by right-clicking on the folder, going to Services at the bottom of the menu, and clicking "New Terminal at Folder". Alternatively, you can type "cd " and drag the folder to the terminal to copy the folder path.

If you use the method of dragging the folder into the terminal, make sure you first type "cd" and then a space before you drag it in.

(fyi, I'm not sure what's going on with your font settings but this is incredibly difficult to read)
